app/src/main/java/com/shakenbeer/nutrition/ui/DayListFragment.java

```diff
@@ -55,6 +56,10 @@
     private DataCursor<Day> dayCursor;
     private ProgressDialog dialog;
 
+    private ListView mListView;
```

Comment:
No need in this field. We use ListFragment here, it has getListView method and it works, because we use @android:id/list.